block-BHaabh/index.html

```diff
@@ -0,0 +1,139 @@
+<!-- 
+### Create a page according to the layout shown below
+
+![Positioning Content Exercise 2](https://raw.githubusercontent.com/suraj122/AC-STYLE-images/master/positioning-content/ex-2.png)
+
+- Using CSS resets is necessary.
+- Use semantic tags and keep the nesting and indentation proper. -->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>website template by TEMPLATED</title>
+    <link rel="stylesheet" href="assets/stye.css">
+    
+</head>
+<body>
+    <header>
+         <div class="container">
+            <a href="#"> <h2  class="logo">Theory </h2></a>
+            <nav class="nav" >
+            <a href ="#">Home</a>
+            <a href ="#">Generic</a>
+            <a href ="#">Elements</a>
+            </nav>
+              <h1 class="heading">Welcome to Theory</h1>
+             <h4> free responsive HTML5 website template by TEMPLATED.</h4>
+         
+          </div>
+        </header>
+        <main>
+            
+         <section class="section">
+             <div class="container clearfix">
+                   <div class="col col-1">  
+                      <h3>Magna tempus sed amet <br>
+                          aliquam veroeros
+                      </h3><br>
+                      <p class="paragraph">Morbi interdum mollis sapien. Sed ac risus.<br> lacinia, magna a ullamcorper<br> laoreet, lectus arcu.
+                      </p>  
+                      <button class="btn"> <a class="more" href="#">More</a></button>
+                   </div>
+                   <div class="col col-2">  
+                      <h3>Interdum lorem pulvinar <br>
+                           adipiscing vitae
+                      </h3> <br>
```

Comment:
Wha's the purpose of <br> tag here. If you are using br tag to provide a gap between elements, then use margin or padding.